citi-onboarding/CESC-client
# Feature/welcome page

##What I did

Created the welcome section
How to test
Open the terminal and go to the folder of this project

In terminal:

git checkout develop
git pull
git checkout feature/welcome_page
git pull origin feature/welcome_page
yarn install
yarn start
In your browser go to http://localhost:3000/

In VScode open pages - Welcome

Test the things that you think are worth testing, even the ones that are not in this description

Make sure the Welcome page is similar to the mockup
